src/database/migrations/20230307201538-create-project.js

```diff
@@ -31,6 +31,10 @@ module.exports = {
       endDate: {
         type: Sequelize.DATE,
       },
+      status: {
+        type: Sequelize.ENUM("pending", "in-progress", "completed"),
```

Comment:
Thank you @Patricknshimiyimana, but I think only in-progress and completed are enough. unless you think there would be cases where project could be in pending mode.

Reply:
Nice, @Patricknshimiyimana, it is totally understandable I get it, we need all three options. like when the project start date has not been yet reached it should in pending mode.